pontoon/base/signals.py

```diff
@@ -17,3 +21,71 @@ def project_locale_added(sender, **kwargs):
         project = project_locale.project
         project.has_changed = True
         project.save(update_fields=['has_changed'])
+
+
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    instance = kwargs['instance']
+
+    if kwargs['raw'] and getattr(instance, 'translators_group', False):
+        return
+
+    locale_ct = ContentType.objects.get(app_label='base', model='locale')
```

Comment:
You do a lot of `get`ing in these signals. If any of the groups accidentally get deleted from the database, we will lose our ability to save anything to database across the entire site.

It'd be nicer if we caught these errors and sent them to RayGun while continuing on. That way we'd be alerted to the error but the site functionality degrades in case of an issue instead of breaking completely.